tests/cross/src/test/scala/tests/hover/RangeHoverSuite.scala

```diff
@@ -96,7 +102,9 @@ class RangeHoverSuite extends BaseHoverSuite {
       "2.13" ->
         """|IndexedSeq[Int]
            |override def flatMap[B](f: Int => IterableOnce[B]): IndexedSeq[B]""".stripMargin.hoverRange,
-      "3" -> "x: Int".hoverRange
+      "3" ->
+        """|IndexedSeq[Int]
+           |def flatMap[B](f: A => IterableOnce[B]): IndexedSeq[Int]""".stripMargin.hoverRange
```

Comment:
So in all of the cases the return type is alright, but I haven't managed to figure out how to change the actual function params. I think we can work on it further on.